generator/metaline_builder.cpp

```diff
@@ -0,0 +1,217 @@
+#include "generator/metaline_builder.hpp"
+#include "generator/routing_helpers.hpp"
+
+#include "indexer/classificator.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/varint.hpp"
+#include "coding/write_to_sink.hpp"
+
+#include "base/checked_cast.hpp"
+#include "base/logging.hpp"
+#include "base/string_utils.hpp"
+
+#include "defines.hpp"
+
+#include <cstdint>
+
+namespace
+{
+using Ways = std::vector<std::int32_t>;
```

Comment:
Why?
In `ctsdint` it is with prefix. The same in our `std/*.hpp` libraries. The only library that defines it without prefix is `stdint`, which is not used in our project.